Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -1171,11 +1171,83 @@ public static bool DidGamepadDisconnectThisFrame(GamepadSlot slot)
             return s_GamepadsDisconnectedFrames[(int)slot] == Time.frameCount;
         }
 
-#if false
-        public static Vector2 mousePosition { get; }
-        public static bool mousePresent { get; }
-        public static float mouseScrollDelta { get; }
-#endif
+        /// <summary>
+        /// The pixel position of the pointer in window space.
+        /// </summary>
+        /// <returns>The position of the first pointer device installed in the system.</returns>
+        /// <remarks>
+        /// If there are multiple pointer devices attached to the system, this will return the pointer
+        /// position of the first one. In window space coordinates, the bottom left corner of
+        /// the window is 0, 0.
+        /// pointerPosition will work for mouse, pen, and touchscreen devices.
+        /// Note that for Touchscreen devices, this will return the position of the primary touch.
+        /// </remarks>
+        public static Vector2 pointerPosition
+        {
+            get
+            {
+                foreach (var inputDevice in InputSystem.devices)
```

Comment:
Wonder if `Pointer.current.position.ReadValue()` could be better here? Because if I have a mouse and a pen attached, then I move first mouse, and then pen, shouldn't I see updated position in both cases? As they move same physical cursor on the screen.